__tests__/index.test.js

```diff
@@ -154,3 +154,57 @@ describe('Bad routes', () => {
     });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+    test('GET 201: accepts request body object with username and body and responds with the posted comment', () => {
+        const newComment = {
+            username: 'mallionaire',
+            body: 'I enjoy playing this game every time and wit the usual suspects'
+        };
+        return request(app)
+            .post('/api/reviews/2/comments')
+            .send(newComment)
+            .expect(201)
+            .then(response => {
+                expect(response.body.insertedComment).toMatchObject({
+                    comment_id: 7,
+                    body: 'I enjoy playing this game every time and wit the usual suspects',
+                    review_id: 2,
+                    author: 'mallionaire',
+                    votes: 0,
+                    created_at: expect.any(String)
+                });
+            });
+
+    });
+    test('POST: 400 responds with appropriate and error message when passed an invalid id', () => {
+        return request(app)
+            .post("/api/reviews/shadyPoint/comments")
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Bad Request");
+            });
+
+    });
+    test('POST: 404 responds with appropriate and error message when passed a valid but non-existent id', () => {
+        return request(app)
+            .post('/api/reviews/777/comments')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Review not Found");
+            });
+
+    });
+    test('POST: 404 responds with appropriate and error message when username is not found', () => {
+        const newComment = {
+            username: 'Keyser Soze',
+            body: 'I enjoy playing this game every time and wit the usual suspects'
+        };
+        return request(app)
+            .post('/api/reviews/2/comments')
+            .send(newComment)
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Username not Found");
+            });
+    });
+});
```

Comment:
Add another test for missing required files e.g. no username or body properties - can test for entirely missing key of username/body, and/or entirely missing property.